conf/etc/v2/components/csm.json

```diff
@@ -0,0 +1,161 @@
+{
```

Comment:
@ajaysrivas Not sure we can put these test related files here. Please let us know your thoughts.

Reply:
@indrajitzagade , @ajaysrivas the thing is that they are not supposed to be test files. Though, I use them in tests.
I tried to fill required information in component json files with some limitations regarding locations and dependencies.

Reply:
ok